CHANGELOG.md

```diff
@@ -1,12 +1,12 @@
-## 19.0.0 (December 5, 2024\)
+## 19.0.0 (December 5, 2024)
```

Comment:
`\` did not exist in other headings, so I removed it (not sure what this escaping would be useful for)

https://github.com/facebook/react/blob/ea353ec81ce6dced67fcdb9261cad00a9783ac32/CHANGELOG.md#L270-L274

Reply:
I saw this in other places as well. Might've been an artifact from copying the original doc into markdown.